### ISSUE TITLE
Disappearing elements' content

### ISSUE THREAD

**Issue by saranvdev** [labyrinth-team/labyrinth#29]
Hi! Is the development active again?

I remember there was a bug that caused elements' content to disappear. The contents were correct but after I save the file and load the map again, some of the elements are empty text. Possibly related to ghost nodes?

**Comment by takluyver** [comment]
> Is the development active again?

Not really, tbh. I got a bit closer to getting it working on Gtk 3, but ran into [an issue with Pango](https://gitlab.gnome.org/GNOME/pango/-/issues/476) again. And I'm not trying to use Labyrinth, so I have little motivation to spend time on it.

**Comment by harcesz** [comment]
Wow, I'm really surprised anybody is even looking at it. My fav mindmapper since years, even with the extra hassle of installing it on new systems nowadays. If there's any point in it I can document some bugs, would be happy to do some testing as required and might have some recommendations as well. 

I really believe in a minimalist mindmap that nicely ties into the rest of the environment.

**Comment by l3td33r** [comment]
> Wow, I'm really surprised anybody is even looking at it. My fav mindmapper since years, even with the extra hassle of installing it on new systems nowadays. If there's any point in it I can document some bugs, would be happy to do some testing as required and might have some recommendations as well.
> 
> I really believe in a minimalist mindmap that nicely ties into the rest of the environment.

welp I just use Heimer instead, looks like a nice, minimalistic replacement

**Comment by harcesz** [comment]
>welp I just use Heimer instead, looks like a nice, minimalistic replacement

doesn't seem like it has multiple connections between nodes, which is important for me, nor the instant 'point and click' creation.

**Comment by takluyver** [comment]
I've just done a bit more work towards GTK 3 compatibility on the [gtk3 branch](https://github.com/labyrinth-team/labyrinth/tree/gtk3). It's getting closer, but still not really usable.

If you want to try this branch, you'll need to build & install my [pango_attr_cast](https://gitlab.gnome.org/takluyver/pango-attr-cast) library using meson - sorry about this, but I ran into a roadblock with Pango and this was the only way I could find to move forwards. Once it's basically working, I'll put it as a beta on Flathub so it's easier to try out.

I've still got Labyrinth open in my IDE for now, but it's something that's fun to fiddle with occasionally, never really a priority. So I might carry on tomorrow, or it might be months/years before I find time again. In principle, I'd welcome other people working on it, but reviewing code changes is not much less work than making them (and it is less *fun* :wink: ), so I can't promise that PRs will actually go anywhere fast.
